adapters/limelightDigital/LimelightDigitaltest/supplemental/empty_object reponse.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Nitpick - missing underscore ("_") in the filename. Recommend renaming it as :-
empty_object_response.json

Reply:
Sorry. I fixed it in https://github.com/prebid/prebid-server/pull/2539/commits/a62aea11c7a63b52f404abd5cd74592861825fb3 commit